clusterloader2/testing/density/config.yaml

```diff
@@ -15,7 +15,7 @@
 # This results in decreased probability of choosing the same node again.
 {{$LATENCY_POD_CPU := DefaultParam .LATENCY_POD_CPU 100}}
 {{$LATENCY_POD_MEMORY := DefaultParam .LATENCY_POD_MEMORY 350}}
-{{$MIN_LATENCY_PODS := 500}}
+{{$MIN_LATENCY_PODS := DefaultParam .MIN_LATENCY_PODS 500}}
```

Comment:
why is this needed?

Reply:
Even though there exists no such override in-tree or in this PR, making this overridable makes it easier to test multiple cl2 executions quickly without having to wait for 3000 saturation pods / 500 latency pods.

Reply:
Given that it's not used anywhere here it's a bit "yagni" to me. 
I'm fine with submitting this if you need it for manual testing, but in that case let's move it to a separate PR

Reply:
Done: https://github.com/kubernetes/perf-tests/pull/1467